test/test_framework.py

```diff
@@ -1235,150 +1233,147 @@ def test_mutable_types(self):
             None,
             {},
             lambda a, b: None,
-            lambda res, expected_res: assert res == expected_res
+            lambda res, expected_res: pytest.fail() if res != expected_res else None
```

Comment:
It seems likely `pytest.fail()` won't include any useful information in the message, which will make future test debugging hard. Would the following work, or is pytest not clever enough to introspect through this:

```
def _assert(cond):
    assert cond
...
lambda a, b: None,
lambda res, expected_res: _assert(res == expected_res),
...
```

Can you think of another way to get reasonable test output if it's different here? We could put an explicit reason in like `pytest.fail(f"{res} != {expected_res}")`, but that would be verbose...

Reply:
I think lambda is a temporary solution. How about I revert this part back to self.assert, then in the next phase, I can update test cases with lambda functions to pytest paramitrize?

On pytestify, I commented out fixers that I don't use and used it from a local directory. I did not add switches/options, nor did I contribute back to the upstream. The upstream seems not maintained for some time.

Reply:
> How about I revert this part back to self.assert, then in the next phase, I can update test cases with lambda functions to pytest paramitrize?

That sounds good to me -- thanks.

> On pytestify, I commented out fixers that I don't use and used it from a local directory. I did not add switches/options, nor did I contribute back to the upstream. The upstream seems not maintained for some time.

Okay, if that's the only change, that's fine.